samples/embedded/embedded.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import os
+
+import functorch
+from functorch._src.compile_utils import strip_overloads
+import iree_torch
+import numpy as np
+from sklearn.datasets import make_regression
+from sklearn.model_selection import train_test_split
+import torch
+import torch_mlir
+
+
+# Dataset
+X, y, coef = make_regression(n_features=3, coef=True)
+X = torch.from_numpy(X).to(dtype=torch.float32)
+y = torch.from_numpy(y).to(dtype=torch.float32)
+X, X_test, y, y_test = train_test_split(X, y)
```

Comment:
Can `X` be lowercase like the other variables? (Here and in the .c file)
I'm sure there is some convention from Python / ML, but consistent casing is nice :P

Reply:
This is really common in ML, to the point I'm not sure I've ever seen it written any other way.

Reply:
Fair enough, thanks for confirming.